app/src/common/utils/getCurrentUrlParam.js

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright 2019 EPAM Systems
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export const getCurrentUrlParam = (param) => {
+  let currentParams = window.location.hash;
+  currentParams = currentParams.substring(currentParams.indexOf('?'));
+  const queryParams = new URLSearchParams(currentParams);
+  return queryParams.get(param);
+};
```

Comment:
We have selector in page controller called `querySelector` which already exist and do the same thing. Please use it if it's fit